gnosis/safe/tests/test_safe.py

```diff
@@ -815,3 +815,21 @@ def test_send_previously_approved_tx(self):
 
         balance = self.w3.eth.get_balance(to)
         self.assertEqual(value, balance)
+
+    def test_get_contracts_cache_leak(self):
+        safe_deployed = self.deploy_test_safe(
+            owners=[self.ethereum_test_account.address]
+        )
+        Safe.get_contract.cache.clear()
+        for i in range(1, 2):
```

Comment:
Maybe I'm not getting this, but this `range(1, 2)` runs just one iteration of the loop. The idea is to assure that while having N calls to `safe.get_contract()` the cache size stays at 1?

Reply:
Yes! This should starts from 0. Thanks